q2_dada2/plugin_setup.py

```diff
@@ -41,6 +41,9 @@
                 'trim_left': qiime2.plugin.Int,
                 'max_ee': qiime2.plugin.Float,
                 'trunc_q': qiime2.plugin.Int,
+                'chimera_method': qiime2.plugin.Str,
+                # IS qiime2.plugin.Str THE RIGHT TYPE?
```

Comment:
Yup!

This can be:
```python
qiime2.plugin.Str % qiime2.plugin.Choices({'pooled', 'consensus', 'none'})
```

We also now have better support for `qiime2.plugin.Range` for the other number primitives in the future.